### PR TITLE
Reduce general binding binary size

### PR DESCRIPTION
**Pull request by baude**
when using the bindings to *only* make a connection, the binary was
rough 28MB.  This PR reduces it down to 11.  There is more work to do
but it will come in a secondary PR.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/errorhandling/errorhandling.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package errorhandling
 
 import (
+	errors2 "errors"
```
Comment (mheon) [inline pkg/errorhandling/errorhandling.go:4]: Superfluous import - the existing `github.com/pkg/errors` package will work identically.

**Comment by rhatdan** [comment]
LGTM other then @mheon comment and the test failures.

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests and @mheon's comment is addressed.

**Comment by rhatdan** [comment]
BTW did this cause podman-remote to shrink?

**Comment by rhatdan** [comment]
Needs a rebase to get the fixes for CI.

**Comment by baude** [comment]
rebased and fixed comment

**Pull request merged**